app/models/rule_group.rb

```diff
@@ -30,4 +32,10 @@ def self.from_openscap_parser(op_rule_group, benchmark_id: nil, parent_id: nil)
 
     rule_group
   end
+
+  def children_with_relationships(requires, conflicts)
+    rules.map do |r|
```

Comment:
Sorry for being too nitpicky, but if its children, should it also not include groups? Can we rename it to `rules_with_relationships` to have it correctly?